metecho/api/models.py

```diff
@@ -230,9 +230,9 @@ def delete(self, *args, **kwargs):
         super().delete(*args, **kwargs)
 
     @property
-    def github_id(self) -> Optional[str]:
+    def github_id(self) -> Optional[int]:
```

Comment:
@jerivas I think this means that `ScratchOrg.owner_gh_id` should become an `IntegerField`, and the migration should do that casting for existing orgs.

Reply:
Fixed in 2e23eb5b1d5fa43b1a9d096ce5215c4c72a5528a.

I wonder if it would be better to turn these fields into a FK? We should have a reference to an actual GHUser object at all times now when creating orgs

Reply:
Oh, good point. Actually we already have `ScratchOrg.owner` which is a FK to the `User` -- could we use that to replace `owner_sf_username`, `owner_gh_username` and `owner_gh_id` and just expose them via the serializer?

Reply:
Yeah, that should be doable. We don't support anyone who isn't an actual Metecho user owning an org, right?

Reply:
Right.